Documents/UseCasesForWebsite.txt

```diff
@@ -1,4 +1,26 @@
--  user wants to see the info page 
--  user wants to apply as artist 
-- user wants to donate 
--  user wants to have a prostictic worked on
\ No newline at end of file
+Actors: Visitor, Artist, Admin
+
+Visitor - View info page, View Gallery, Donate, RequestProstetic, Liking a Picture.
+A visitor would use this website for geting information or donate as amputee a prostetic or donate as a company.Also the visitors can visit the gallery and like one of the displayed art there.
+A visitor 
+- can also request that his/her prostetic is send to artist to make some art of it.
+- to donate a protestic he/she fill in a form and the admin of the site would contact the visitor. 
+- who is representing a company wants to donate a prostetic , also a form is filled in. 
+- goto the gallery page and like a prostic which is turn into art by a artist, he/she can like it so the prostetic is displayed earlier on the gallery page. 
+
+Artist - Apply to be part of this project so the artist can paint a prostetic.
+An artist could use this website for the same reasons as a regular user, but also the artist can fill in a form and send two pictures of art the artist make so the admin can look if the artist is good enough to 
+paint a prostetic for a amputee or for the fun raising. 
+-  a artist can apply to be a part of the project by filling in a apply form , send in 2 images of work he/she made and gets approved or disapproved.
+
+Admin - Manage Galleries. Manage request for artist or request from a amputee
+An admin would use this website to manage the open gallery, accept  or decline incoming pictures that people want to post to the promotional gallery (kind of a moderator role), and they could also manage artists as well.
+A admin can 
+-  give an artist wants has applied to become a "Professional Artist" on the site/app the right to become a  "Proffesional Artist".
+- Managing reques from a amputee means that  the admin sees the request and checked with a company who makes the prostetics if this is a real request. Also the admin sends a messages if the amputee or artist is chosen
```

Comment:
This is actually 2 use cases:
**View amputee requests**- specify what exactly is a proof that company checked the person (doc, code, etc).
**Approve amputee request**- what happens with the approved request (archived or deleted or kept?). How does notification to amputee happen? Can requests expire? 

I also personally think that the proof for amputee legitimacy is mandatory, therefore it should not be possible declining a request.Maybe a better word is hiding? Just food for thought.

Reply:
oke, here I have to think about

Reply:
What do you exactly mean with hiding, I agree with you that declining is not a option.
So we have as statusses then :  recieved, send to artist, send to requester, done. 
and I think a request can never be expired.

Reply:
Imagine there are 500 requests. Do you want to see all? Maybe instead of hiding, we can have a priority request flag?

Reply:
oke, then im thinking about using pagination.  We could use a priority flag. I will think of the conditions when a request get which priority.